docs/StartHere.md

```diff
@@ -0,0 +1,9 @@
+# Tidepool uploader developer guide
+
+As you're getting ready to develop code in this repository, we recommend starting with the following documents:
```

Comment:
They are parallel to what I've done for blip and viz to varying levels of completeness. How parallel we want to be across repos is definitely an open question.